tests/components/fritz/const.py

```diff
@@ -26,9 +26,758 @@
     }
 }
 MOCK_HOST = "fake_host"
-MOCK_IP = "192.168.178.1"
+MOCK_IPS = {"fritz.box": "192.168.178.1", "printer": "192.168.178.2"}
+MOCK_MODELNAME = "FRITZ!Box 7530 AX"
+MOCK_FIRMWARE = "256.07.29"
 MOCK_SERIAL_NUMBER = "fake_serial_number"
 MOCK_FIRMWARE_INFO = [True, "1.1.1"]
+MOCK_MESH_SSID = "TestSSID"
+MOCK_MESH_MASTER_MAC = "1C:ED:6F:12:34:11"
+MOCK_MESH_MASTER_WIFI1_MAC = "1C:ED:6F:12:34:12"
+MOCK_MESH_SLAVE_MAC = "1C:ED:6F:12:34:21"
+MOCK_MESH_SLAVE_WIFI1_MAC = "1C:ED:6F:12:34:22"
+
+MOCK_FB_SERVICES: dict[str, dict] = {
```

Comment:
Normally we put this kind of data in json files in the integration fixtures test directory. Then load that in a session scoped pytest fixture.

Reply:
Sounds very reasonable, thanks for this hint, will add this in the upcoming improvement PRs 👍